modules/misunderstood/src/backend/db.ts

```diff
@@ -47,15 +47,18 @@ export default class Db {
     await this.knex(TABLE_NAME).insert(event)
   }
 
-  async updateStatus(botId: string, id: string, status: FLAGGED_MESSAGE_STATUS, resolutionData?: ResolutionData) {
+  async updateStatuses(botId: string, ids: string[], status: FLAGGED_MESSAGE_STATUS, resolutionData?: ResolutionData) {
     if (status !== FLAGGED_MESSAGE_STATUS.pending) {
       resolutionData = { resolutionType: null, resolution: null, resolutionParams: null }
     } else {
       resolutionData = pick(resolutionData, 'resolutionType', 'resolution', 'resolutionParams')
     }
 
     await this.knex(TABLE_NAME)
-      .where({ botId, id })
+      .where({ botId })
+      .andWhere(function() {
+        this.whereIn('id', ids)
```

Comment:
I don't know what's the extent of the this module's usage but this will eventually fail if there are too many ids (like 100 ) especially when using sqlite which has a query length constraint

Reply:
Discussed offline: the SQL statement length limit for SQLite is high enough (1,000,000,000 bytes) that this will be a non-issue